pkgs/development/python-modules/trustme/default.nix

```diff
@@ -11,6 +11,11 @@
 , idna
 }:
 
+let
+  isAarch64Darwin = stdenv.isDarwin && stdenv.isAarch64;
+
+in
+
```

Comment:
```suggestion
```
Despite this being longer we usually do not add extra variables.

Reply:
(Length was less of a concern than triplication.)